pyports/cluster_activities_utils.py

```diff
@@ -95,8 +126,9 @@ def polygenize_clusters_with_features(type_of_area_mapped, df_for_clustering, po
         else:
             polygon = polygon_from_points(points, polygon_type, alpha)  # create polygon from points
 
-        record['label'] = f'cluster_{cluster}'  # cluster label
-
+        record['label'] = f'cluster_{int(cluster)}'  # cluster label
+        if type_of_area_mapped == 'pwa':  # get destination port name
+            record['destination_port'] = cluster_df['nextPort_name'].mode()[0]
         record['probs_of_belonging_to_cluster'] = \
             ', '.join(cluster_df['cluster_probability'].astype(str).to_list())  # list of all points probabilities of
         # belonging to the cluster
```

Comment:
Are all the record features still relevant? do you eventually use all of them? or some of them were turned out to be non relevant? If there are such - please omit them